cpp/basix/finite-element.h

```diff
@@ -725,24 +754,30 @@ class FiniteElement
            std::vector<std::tuple<std::vector<std::size_t>, std::vector<double>,
                                   xt::xtensor<double, 2>>>>
       _etrans_invT;
+
+  /// Indicates whether or not this is the discontinuous version of the element
+  bool _discontinuous;
 };
 
 /// Create an element using a given lattice type
 /// @param[in] family The element family
 /// @param[in] cell The reference cell type that the element is defined on
 /// @param[in] degree The degree of the element
 /// @param[in] lattice_type The lattice type that should be used to arrange DOF
-/// points of the element
-FiniteElement
-create_element(element::family family, cell::type cell, int degree,
-               lattice::type lattice_type);
+/// @param[in] discontinuous Indicates whether the element is discontinuous
+/// between cells points of the element
+FiniteElement create_element(element::family family, cell::type cell,
+                             int degree, lattice::type lattice_type,
+                             bool discontinuous = false);
 
 /// Create an element
 /// @param[in] family The element family
 /// @param[in] cell The reference cell type that the element is defined on
 /// @param[in] degree The degree of the element
-FiniteElement
-create_element(element::family family, cell::type cell, int degree);
+/// @param[in] discontinuous Indicates whether the element is discontinuous
+/// between cells
+FiniteElement create_element(element::family family, cell::type cell,
+                             int degree, bool discontinuous = false);
```

Comment:
In my experience default arguments like this can be quit dangerous because `int` can be silently cast to `bool`. User could make a mistake to the arguments and it won't het picked up by the compiler.